packages/tizen_app_manager/tizen/src/tizen_app_manager.h

```diff
@@ -21,12 +21,12 @@ class TizenAppManagerPlugin : public flutter::Plugin {
 
   virtual ~TizenAppManagerPlugin();
 
-  flutter::EncodableList m_applications;
-  std::unique_ptr<flutter::EventSink<flutter::EncodableValue>> m_launch_events;
+  flutter::EncodableList applications_;
+  std::unique_ptr<flutter::EventSink<flutter::EncodableValue>> launch_events_;
   std::unique_ptr<flutter::EventSink<flutter::EncodableValue>>
-      m_terminate_events;
-  bool m_registered_event_cb;
-  int m_registered_cnt;
+      terminate_events_;
+  bool registered_event_cb_;
```

Comment:
suffix `cb` is typically used when a variable is a function pointer used as a callback. please consider renaming.

Reply:
I agree that it is inappropriate to put cb as a suffix.
In addition, we will change it to has_registered_event_ to clearly convey what the variable means.